src/main/java/sparrow/commands/FilterCommand.java

```diff
@@ -0,0 +1,36 @@
+package sparrow.commands;
+
+import sparrow.data.task.Task;
+import sparrow.data.task.TaskList;
+import sparrow.storage.Storage;
+import sparrow.ui.Ui;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class FilterCommand extends Command {
+
+    public static final String COMMAND_WORD = "filter";
+
```

Comment:
Nice that the constants are properly labelled: public static final followed by an all caps name.